### ISSUE TITLE
[Feature Request]: "Filed under" field

### ISSUE THREAD

**Issue by themerekat** [Symbiota/community-board#155]
### Symbiota Portal Name

CCH2

### Is your feature/tool request related to a problem? Please describe.

From Mare Nazaire: "I am wondering if it has ever come up with other CCH members or if it is possible to add a field in the portal under the Scientific Name field: "Filed as" or "Filed under" - because we frequently come up against this issue in our collection where we have not yet updated our taxonomy in the collection and it may be some time before we can get to it all. It would be helpful to have a field like this so that we could indicate where a specimen can be found if the taxonomy has not been updated. It could also use the taxon thesaurus, and it would also be helpful to have that field when capturing skeletal data during our imaging process. Is that possible? I just thought I would ask."

**Comment by mickley** [comment]
We also have this issue and have been using [dwc:disposition](https://dwc.tdwg.org/list/#dwc_disposition) for this. I am not sure if this is the ideal, though.

**Comment by themerekat** [comment]
Mentioned by the SERNEC community in a webinar today

**Comment by STHerbariumLSU** [comment]
I agree that this field is highly desirable.  It would also be nice to have this function in some type of batch mode, so that many names can be linked to a filed as location when returning to records to add this information.  Additionally, it would be nice to have a taxon name dropdown list built into the FiledAs field so that there is less user error.  Thanks!

And I agree that this can somehow be expanded to include basionym information.

**Comment by themerekat** [comment]
> We also have this issue and have been using [dwc:disposition](https://dwc.tdwg.org/list/#dwc_disposition) for this. I am not sure if this is the ideal, though.

This is likely the best place to put this information now.

**Comment by themerekat** [comment]
- [ ] Make field
- [ ] Link to taxonomic thesaurus (for auto-populating, but not recording tids)
- [ ] Make batch editable
- [ ] Make searchable through Record Search Form

**Comment by mickley** [comment]
> * [ ]  Make field[ ]  Link to taxonomic thesaurus (for auto-populating, but not recording tids)[ ]  Make batch editable[ ]  Make searchable through Record Search Form

@themerekat: there are potential uses that don’t involve taxonomic names, as well.  For example, filed under xxx collection (which is filed separately): a separate herbarium, boxed specimens, spirit collections, etc.

**Comment by themerekat** [comment]
> > * [ ]   Make field[ ]  Link to taxonomic thesaurus (for auto-populating, but not recording tids)[ ]  Make batch editable[ ]  Make searchable through Record Search Form
> 
> [@themerekat](https://github.com/themerekat): there are potential uses that don’t involve taxonomic names, as well. For example, filed under xxx collection (which is filed separately): a separate herbarium, boxed specimens, spirit collections, etc.

@mickley , those definitely fall under "disposition" then, imo
